internal/distro/rhel8/azure.go

```diff
@@ -571,3 +576,33 @@ var azureRhuiImgType = imageType{
 	exports:             []string{"archive"},
 	basePartitionTables: azureRhuiBasePartitionTables,
 }
+
+func SapAzureImageConfig(rd distribution) *distro.ImageConfig {
+	return SapImageConfigX86_64(rd).InheritFrom(defaultAzureImageConfig)
+}
+
+// TODO What to put in here
+func azureSapImgType(rd distribution) imageType {
+	return imageType{
+		name:     "azure-sap",
```

Comment:
Is this image supposed to include RHUI client packages? If yes, I propose to call the image type `azure-sap-rhui`. We should have probably done this for EC2 images in the beginning. The proposed name in case of RHUI is consistent with image naming used by Cloud API and IMO it makes sense to stick to it if possible.